packages/cli/src/databases/entities/AbstractEntity.ts

```diff
@@ -45,17 +40,13 @@ function mixinStringId<T extends Constructor<{}>>(base: T) {
 
 function mixinTimestamps<T extends Constructor<{}>>(base: T) {
 	class Derived extends base {
-		@CreateDateColumn(tsColumnOptions)
-		@IsOptional() // ignored by validation because set at DB level
-		@IsDate()
+		@Column(tsColumnOptions)
 		createdAt: Date;
 
-		@UpdateDateColumn({
+		@Column({
 			...tsColumnOptions,
 			onUpdate: timestampSyntax,
```

Comment:
is this `onUpdate` trigger really needed? is the `@BeforeUpdate()` section below not enough?

Reply:
The docs for this option say: `ON UPDATE trigger. Works only for MySQL.` and based on examples I'm finding it seems to take directives like `CASCADE` and such, not a function like `CURRENT_TIMESTAMP()`, so our current usage is wrong anyway?

The docs for `@UpdateDateColumn()` say `Special column that is automatically set to the entity's update time each time you call save from entity manager or repository.`  which I assume is not good enough since we sometimes update using methods other than `save()`. 

So it looks like we have to keep the workaround below but we can remove this option here.

Reply:
had to put this back for mysql 🤷🏽